### DISCUSSION TITLE
Sequential single scenario execution

### DISCUSSION THREAD

**Discussion by michaelplavnik** [VIP-Logic/NBomber#712] [General]
Is it possible to execute single scenario N times in sequence, not concurrently, and without imposed delay between iterations?

Would appreciate any ideas. Also willing to contribute if code changes are needed.

**Comment by AntyaDev** [comment]
Hi @michaelplavnik ,
Yes, please use this LoadSimulation: https://nbomber.com/docs/nbomber/load-simulation#iterations-for-constant
